src/main/java/prefs/UserConfig.java

```diff
@@ -0,0 +1,19 @@
+package prefs;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test class to see if the file handler works with other config subclasses in the new framework
+ */
+public class UserConfig implements Config {
```

Comment:
Yeah it was just as a proof of concept. I can remove this file for now. Private class sounds good, I'll try that method